src/main/java/org/hydev/mcpm/client/database/LocalPluginTracker.java

```diff
@@ -21,11 +24,16 @@
  */
 public class LocalPluginTracker {
     // CSV file storing the list of manually installed plugins
-    private static String MainLockFile = "TODO: Get this path";
+    private String MainLockFile = "TODO: Get this path";
 
     // Directory storing the plugins
-    private static String PluginDirectory = "TODO: Get this path";
+    private String PluginDirectory = "TODO: Get this path";
 
+    // Constructor 
+    public LocalPluginTracker(String MainLockFile, String PluginDirectory) {
```

Comment:
Careful with checkstyle!

```suggestion
    public LocalPluginTracker(String mainLockFile, String pluginDirectory) {
```